tests/dummy/app/templates/usage.hbs

```diff
@@ -14,7 +14,16 @@ To declaratively attach an action to a keyboard event while a particular templat
 
 Note that the listener is active for the lifecycle of the template. When the template is unrendered, the listener is removed.
 
-Another option for templates is the `keyboard-shortcut` element modifier. This allows a keyboard event to simulate clicking the element the modifier is attached to. In the following example, we trigger a click on the button when the B key is pressed together with the Alt key:
+Another option for templates is the `on-keyboard` element modifier. This allows a keydown event to trigger an action. In the following example, we trigger an action when the input field is focused and the user presses enter:
+
+```hbs
+<input type='text'
+  oninput={{action (mut this.textFieldValue) value='target.value'}}
+  {{on-keyboard 'Enter' (action 'onEnterPressedInInput')}}
```

Comment:
Shouldn't this be `(fn this.onEnterPressedInInput)`?

Reply:
It depends on how it is defined in the component, controller, or route, which is not specified here.

Reply:
Clearly I have things to learn...